Wireframe/index.html

```diff
@@ -4,10 +4,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+      <h1>All about Git💻</h1>
+      <p>Git is a tool that helps you keep track of changes to files, especially code. It’s like a "save" button, but smarter! lets you see who changed what and when, and even go back to old versions if needed.</p>
+    </header>
+
+    <main>
+
+      <article>
+        <h2>What is Git?</h2>
+        <img src="./images/git-logo.png" alt="">
+        <p>Git is a tool that helps you track changes to files, especially when you're working on code. Think of it like a super-powered "undo" button that lets you go back to any point in your project’s history. It’s also great for working with others because it lets everyone make changes at the same time without overwriting each other's work. Git is widely used by developers to manage code, but it can be used for any project where you need to keep track of changes.</p>
+        <p>By far, the most widely used modern version control system in the world today is Git. Git is a mature, actively maintained open source project originally developed in 2005 by Linus Torvalds, the famous creator of the Linux operating system kernel.
+        </p>
+        <a href="https://www.atlassian.com/git/tutorials/what-is-git">Read More</a>
+      </article>
+    <div id="second-part">
+      <article>
+        <h2>Why do developers need Git?</h2>
+        <img src="./images/user.png" alt="">
+        <p>For developers, it eliminates everything from the time wasted passing commits over a network connection to the man hours required to integrate changes in a centralized version control system. It even makes better use of junior developers by giving them a safe environment to work in.</p>
+        <a href="https://www.atlassian.com/git/tutorials/why-git#:~:text=Git%20is%20all%20about%20efficiency,safe%20environment%20to%20work%20in.">Read More</a>
+      </article>
+      <article>
+        <h2>What is a branch in Git?</h2>
+        <img src="./images/git-branch.png" alt="">
+        <p>FGit branches are effectively a pointer to a snapshot of your changes. When you want to add a new feature or fix a bug—no matter how big or how small—you spawn a new branch to encapsulate your changes. T</p>
+        <a href="https://www.atlassian.com/git/tutorials/using-branches#:~:text=Git%20branches%20are%20effectively%20a,branch%20to%20encapsulate%20your%20changes.">Read More</a>
+      </article>
+    </div>
+    </main>
+    <footer>
+      To learn more about Git from a pro contact Serna Malala @ sm.sernamalala@gmail.com
```

Comment:
There is a nice touch to the emails in html. You basically wrap your email into anchor tag with mailto link. Have a look and try to implement it if you like.
https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_link_mailto